sdks/java/core/src/main/java/org/apache/beam/sdk/testing/DataflowRunnerV2Incompatible.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.testing;
+
+import org.apache.beam.sdk.annotations.Internal;
+
+/** Category tag for sick-baying validation tests which are incompatible with Dataflow Runner V2. */
+@Internal
+public interface DataflowRunnerV2Incompatible {}
```

Comment:
We should not tag tests with this, but have an exclude list in the build file.

Reply:
Yeah, I agree. By having a list of excluded tests in a single place, we could better manage them in future. I considered that in my first draft but failed to find the way to exclude specific test methods. It looks like Gradle [JUnit plugin](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/testing/junit/JUnitOptions.html) only supports categorical exclusion and the minimal unit of exclusion supported by Gradle [test task](https://discuss.gradle.org/t/how-to-exclude-a-single-test-method/20860/2) is a class (and it's still too coarse).

Do you know any other good solution to this?

Reply:
I think it should work. Here is an example where I did it: https://github.com/apache/beam/blob/c0755cd5cf338115f9bc5a06ef6359772a9cb816/runners/portability/java/build.gradle#L162

Reply:
Oh thanks! I'll try it :smile:

Reply:
done.